x/claim/keeper/hooks.go

```diff
@@ -44,9 +45,15 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 }
 
 func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
-	err := k.ResetClaimStatus(ctx, epochInfo.Identifier)
-	if err != nil {
-		k.Logger(ctx).Error(fmt.Sprintf("failed to reset claim status for epoch %s: %s", epochInfo.Identifier, err.Error()))
+	// check if epochInfo.Identifier is an airdrop epoch
```

Comment:
If an epoch id is prefixed with "airdrop-", it might not be related to an airdrop, but I think this is fine! We might still want to add a quick validation check to be safe though. You can call `GetAirdropByIdentifier` to check if one exists